test/phabricator_tests/mock_phabricator.py

```diff
@@ -0,0 +1,350 @@
+                'css': [],
+                'summary': 'This is a summary.',
+                'statusName': 'Accepted'
+            },
+        ]
+        return res
+
+    def mock_phabricator_get_comments_single(self):
+        return [
+            {
+                'action': 'comment',
+                'authorPHID': 'PHID-USER-xxxxxxxxxxxxxxxxxxxx',
+                'revisionID': '1',
+                'content': 'This is some content',
+                'dateCreated': '1551763640'
+            }
+        ]
+
+    def mock_phabricator_get_comments_many(self):
+        return [
+            {
+                'action': 'comment',
+                'authorPHID': 'PHID-USER-xxxxxxxxxxxxxxxxxxx3',
+                'revisionID': '1',
+                'content': 'This is some content',
+                'dateCreated': '1551763640'
+            },
+            {
+                'action': 'comment',
+                'authorPHID': 'PHID-USER-xxxxxxxxxxxxxxxxxxxx',
+                'revisionID': '1',
+                'content': 'This is some content',
+                'dateCreated': '1551763640'
+            },
+            {
+                'action': 'comment',
+                'authorPHID': 'PHID-USER-xxxxxxxxxxxxxxxxxxxx',
+                'revisionID': '1',
+                'content': 'This is some content',
+                'dateCreated': '1551763640'
+            }
+        ]
+
+    def mock_phabricator_get_revision_comments(self):
+        return [[
+            {
+                'action': 'comment',
+                'authorPHID': 'PHID-USER-xxxxxxxxxxxxxxxxxxx3',
+                'revisionID': '1',
+                'content': 'This is some content',
+                'dateCreated': '1551763640'
+            },
+            {
+                'action': 'action',
+                'authorPHID': 'PHID-USER-xxxxxxxxxxxxxxxxxxxx',
+                'revisionID': '1',
+                'content': 'This is some content',
+                'dateCreated': '1551763640'
+            },
+            {
+                'action': 'comment',
+                'authorPHID': 'PHID-USER-xxxxxxxxxxxxxxxxxxxx',
+                'revisionID': '1',
+                'content': 'This is some more content',
+                'dateCreated': '1551763640'
+            }
+        ]]
+
+    def mock_phabricator_get_comments_return_value(self):
+        return [
+            {
+                'action': 'comment',
+                'authorPHID': 'PHID-USER-xxxxxxxxxxxxxxxxxxx3',
+                'revisionID': '1',
+                'content': 'This is some content',
+                'dateCreated': '1551763640'
+            },
+            {
+                'action': 'comment',
+                'authorPHID': 'PHID-USER-xxxxxxxxxxxxxxxxxxxx',
+                'revisionID': '1',
+                'content': 'This is some more content',
+                'dateCreated': '1551763640'
+            }
+        ]
+
+    def mock_phabricator_get_reviews(self):
```

Comment:
This would be better.
```python
def mock_phabricator_get_reviews(self):
    return [{
        'dateCreated': 'mock_date',
        'dateModified': 'mock_date',
        'id': 0,
        'title': 'mock_title',
        'authorPHID': 'PHID-USER-xxxxxxxxxxxxxxxxxxxx',
        'uri': 'mock.com'
    }]
```